django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
So it should be first? Otherwise, I agree with Aymeric's point that it wouldn't ever be used. If it's required to make the tests run but shouldn't be in the defaults, then please use `@override_settings()` in the tests.